packages/app-page-builder/src/editor/plugins/elements/pagesList/graphql.ts

```diff
@@ -3,19 +3,12 @@ import gql from "graphql-tag";
 export const LIST_PUBLISHED_PAGES = gql`
     query ListPublishedPages(
         $where: PbListPublishedPagesWhereInput
-        $sort: PbListPagesSortInput
         $limit: Int
-        $page: Int
+        $sort: [PbListPagesSort!]
```

Comment:
Good work on this @dibenso  👍 

Please also include the `after` argument because it is essential to implement pagination:

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/13612227/139794840-061d61ae-e80d-4d90-8225-21b72f0bc600.png">